src/main/java/org/mastodon/mamut/tomancak/lineage_registration/LineageRegistrationFrame.java

```diff
@@ -141,6 +150,9 @@ public LineageRegistrationFrame( Listener listener )
 		add( new JLabel( "Copy tag set:" ) );
 		add( newOperationButton( "from A to B ...", COPY_TAGSET_TOOLTIP, listener::onCopyTagSetAtoB ), "split 2" );
 		add( newOperationButton( "from B to A ...", COPY_TAGSET_TOOLTIP, listener::onCopyTagSetBtoA ), "wrap" );
+		add( new JLabel( "Angles between cell division directions:") );
+		add( newOperationButton( "plot angles", PLOT_ANGLES_TOOLTIP, listener::onPlotAnglesClicked ), "split 2" );
+		add( newOperationButton( "add angles to table", ANGLES_FEATURE_TOOLTIP, listener::onAddAnglesFeatureClicked ), "wrap" );
```

Comment:
The button shouldn't stay down. Must be a bug in JFreeChart or Swing, because the code here does return after making the ChartFrame visible. Could you debug this? I cannot reproduce the problem on my PC. Also the GIF didn't upload correctly.

Reply:
Okay. It might be a Windows only problem then. I add the gif again that shows how to reproduce this. I would like to do the debugging together with you (on my machine), since you know better how things are meant to operate with each other in the code.

![GIF 09 06 2023 14-47-49](https://github.com/mastodon-sc/mastodon-tomancak/assets/10515534/fe4ec717-db63-4c20-8457-3b902bdc8633)